site/docs/components/tag/examples.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="tag" exampleName="Primary">
+
+## Primary
+
+Primary tags feature subtle background colors by default, making them ideal for presenting information or content without being overly attention-grabbing.
+
+To add an optional border, simply pass the `bordered` prop to the component.
+
+### Best practices
+
+- Ensure tag colors are consistent with their associated categories.
+- Be mindful that using an excessive number of colors can impose a heavy cognitive load on some users.
+
+</LivePreview>
+<LivePreview componentName="tag" exampleName="Secondary">
+
+## Secondary
+
+Secondary tags are generally used to emphasize or highlight specific content. These tags use a bold background color to draw more attention and make the highlighted content stand out.
+
+</LivePreview>
+<LivePreview componentName="tag" exampleName="Icon">
+
+## Icon
```

Comment:
In Button, we use "Icon and Text", or should it be "With Icon"?

Reply:
I'll use `With Icon`, as we do in pill cc. @shey-v